biodatacatalyst-ui/src/main/webapp/picsureui/api-interface/bdcTerraLink.hbs

```diff
@@ -0,0 +1,34 @@
+<style>
+    #bdc-terra-body ul {
+        list-style-type: none;
+    }
+
+    #bdc-terra-body li {
+        margin: 10px 0;
+    }
+
+    #bdc-terra-body button,
+    #bdc-terra-body a {
+        width: 100%;
+        text-align: left;
+    }
+</style>
+<p class="sub-title">Please select your preferred programming language and platform.</p>
+<div style="margin: 30px auto 0 auto; width: max-content" id="bdc-terra-body">
+    <ul>
+        <li>
+            <a class="btn btn-lg btn-primary" href="https://tinyurl.com/BDC-PIC-SURE-API-Python-Terra"
+               target="_blank" tabindex="0">
```

Comment:
You do not have to change anything, this is an FYI. But when you pass `handleTabs: true` to the modal component it should take care of tabbing if you add the class `tabable` to any element.